app/views/catalog/_home_text.html.erb

```diff
@@ -40,4 +40,7 @@
       <%= content_tag :div, '', id: 'map', aria: { label: t('geoblacklight.map.label') }, data: { map: 'home', 'catalog-path'=> search_catalog_path , 'map-bbox' => params[:bbox], basemap: geoblacklight_basemap, leaflet_options: leaflet_options } %>
     </div>
   </div>
+  <div align='right'>
+  <p>GeoBlacklight version <%= Geoblacklight::VERSION %></p>
```

Comment:
Should this also be internationalized?

Reply:
Thanks for these! Yeah, I am open to any / all suggestions. Was just following that example (more or less) on the test Blacklight page. And yes, the text should be internationalized. We can discuss at the stand-up or later on the the afternoon for me? 

And as for the .DS_Store thing, sure, that can be deleted.